website/project/views/tag.py

```diff
@@ -48,9 +48,37 @@ def project_add_tag(auth, node, **kwargs):
 @must_have_permission('write')
 @must_not_be_registration
 def project_remove_tag(auth, node, **kwargs):
-
     data = request.get_json()
     tag = data['tag']
     if tag:
         node.remove_tag(tag=tag, auth=auth)
         return {'status': 'success'}
+
+
+@must_be_valid_project  # injects project
+@must_have_permission('write')
+@must_not_be_registration
+def file_add_tag(auth, node, guid, **kwargs):
+    data = request.get_json()
+    tag = data['tag']
+    file_name = data['fileName']
+    if tag:
+        try:
+            fileobject = Guid.load(guid).referent
+            fileobject.add_tag(tag, auth, node, file_name)
+            return {'status': 'success'}, http.CREATED
```

Comment:
This line should also be removed from the `try` block. As this is the `return` statement that you expect to be executed, the last line of the method is the right place for it :)